radiation/python/config.py

```diff
@@ -1,14 +1,20 @@
 import numpy as np
 import sys
 
-sys.path.insert(
-    0, "/Users/AndrewP/Documents/work/physics_standalone/radiation/python/radlw"
-)
+sys.path.insert(0, "/work/radiation/python/radlw")
```

Comment:
this is great because it works in Docker now, but it won't work on your machine anymore, right? It's be nice to reorganize the code such that you don't need to insert a path in each file. But this can be a good stepping stone PR

Reply:
It doesn't, but I've been working entirely in Docker now, so that this is reproducible. Once this is called from the main runfile like all the other physics routines we can get rid of these path additions, but we're not quite there yet